src/cpp/TextNoteCollection.cpp

```diff
@@ -46,3 +46,16 @@ void TextNoteCollection::print() const{
 		notes[i].print();
 	}
 }
+
+TextNoteCollection *TextNoteCollection::addFromCLI(const std::string &title) {
```

Comment:
Why return pointer?
For add note methods, it makes (some) sense, because we can call collectionPtr->add(note1)->add(note2);
For CLI, we probably will call this method only once per process execution
If we want to chain these calls, how do we obtain all titles before asking for all texts?

And in any case, you can do chaining with references: collection.add(note1).add(note2);